optuna/integration/botorch.py

```diff
@@ -486,12 +512,14 @@ def __init__(
                     "torch.Tensor",
                     Optional["torch.Tensor"],
                     "torch.Tensor",
+                    Optional["torch.Tensor"],
                 ],
                 "torch.Tensor",
             ]
         ] = None,
         constraints_func: Optional[Callable[[FrozenTrial], Sequence[float]]] = None,
         n_startup_trials: int = 10,
+        consider_running_trials: bool = False,
```

Comment:
My personal opinion is that making `consider_running_trials=True` as default is okay. What do you think, @HideakiImamura ?

Reply:
```suggestion
        consider_running_trials: bool = True,
```

Reply:
I do not have a strong preference for this but it may slow down the parameter suggestion if the number of running trials is large(, though I did not observe any significant slowdown while testing it with the number of running trials up to 10). I will leave it to @HideakiImamura and @contramundum53.

Reply:
I think it's OK to change the default behavior of this sampler since this is still an experimental.